MAUI/Picker/getting-started.md

```diff
@@ -0,0 +1,308 @@
+
+* Create a simple Observable Collection of data for the picker,
+
+{% tabs %}
+{% highlight c# tabtitle="ItemInfo.cs" %}
+
+public class ItemInfo
+{
+    private ObservableCollection<object> dataSource = new ObservableCollection<object>()
+    {
+        "Pink", "Green", "Blue", "Yellow", "Orange", "Purple", "SkyBlue", "PaleGreen"
+    };
+
+    public ObservableCollection<object> DataSource
+    {
+        get
+        {
+            return dataSource;
+        }
+        set
+        {
+            dataSource = value;
+        }
+    }
+
+    public ItemInfo()
+    {
+
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+* Bind the Collection to picker
+
+Picker control allows you to bind the collection of data by setting the [ItemsSource] property from [PickerColumn]. You can bind the collection of data in both XAML or C#.
+
+{% tabs %}
+{% highlight xaml tabtitle="XAML" hl_lines="6 10 11 12" %}
+
+<picker:SfPicker x:Name="picker">
+    <picker:SfPicker.HeaderView>
+        <picker:PickerHeaderView Text="Select a color" Height="40" />
+    </picker:SfPicker.HeaderView>
+    <picker:SfPicker.Columns>
+        <picker:PickerColumn ItemsSource="{Binding DataSource}" />
+    </picker:SfPicker.Columns>
+</picker:SfPicker>
+
+<ContentPage.BindingContext>
+    <local:ItemInfo />
+</ContentPage.BindingContext>
+
+{% endhighlight %}
+{% highlight c# tabtitle="C#" hl_lines="1 8" %}
+
+ItemInfo itemInfo = new ItemInfo();
+SfPicker picker = new SfPicker()
+{
+    Columns = new ObservableCollection<PickerColumn>()
+    {
+        new PickerColumn()
+        {
+            ItemsSource = itemInfo.DataSource,
+        }
+    }
+};
+
+this.Content = picker;
+
+{% endhighlight %}
+{% endtabs %}
+
+
+## Set footer to the Picker
+
+In SfPicker control, validation buttons (OK and Cancel) can be customized by setting the [OkButtonText] and [CancelButtonText] property in the [PickerFooterView]. It allows you to confirm or cancel the selected item.  The `OkButtonText` can be enabled by using the [ShowOkButton] property in [PickerFooterView].
+
+{% tabs %}
+{% highlight xaml tabtitle="XAML" hl_lines="3" %}
+
+<picker:SfPicker x:Name="Picker">
+    <picker:SfPicker.FooterView>
+        <picker:PickerFooterView ShowOkButton="True" Height="40" />
+    </picker:SfPicker.FooterView>
+</picker:SfPicker>
+
+{% endhighlight %}
+{% highlight c# tabtitle="C#" hl_lines="4 5" %}
+
+SfPicker picker = new SfPicker();
+picker.FooterView= new PickerFooterView()
+{  
+    ShowOkButton = true,
+    Height = 40,
+};
+
+this.Content = picker;
+
+{% endhighlight %}  
+{% endtabs %}
+
+## Set column header to the Picker items
+
+The SfPicker control allows you to the add the column header text by setting the [HeaderText] property in [PickerColumn] and you need to add the column header height by setting the [Height] property in [PickerColumnHeaderView]. Its text style and background color are customizable.
```

Comment:
The SfPicker control allows you to add the column header text by setting the [HeaderText] property in the [PickerColumn], and you need to add the column header height by setting the [Height] property in the [PickerColumnHeaderView]. Its text style and background color are customizable.